docs/malware-scanner/README.md

```diff
@@ -1 +1,44 @@
-## *Currently empty, check back later or contribute yourself*
\ No newline at end of file
+# Malware scanner and detector
+
+---
+
+# Dynamic analysis 
```

Comment:
Since static always happens, and dynamic is opt-in depending on usage, I think it'd make more sense for this section to come after the static one